src/components/Dashboard/Dashboard.js

```diff
@@ -70,7 +78,27 @@ function Dashboard() {
         <Welcome />
       )}
 
-      <Tasks onlyTable />
+      <Split
+        primary={<Tasks onlyTable />}
+        secondary={
+          connectedAccount && (
```

Comment:
Should we provide an empty state, or remove the `Split` when the user is not connected? So that it doesn’t create an empty space on the left side.

Reply:
Ah yes, good call! Until we don't have the dispute total status on the dashboard i think it makes sense